### ISSUE TITLE
v3.3.0 branch same name as v3.3.0 tag

### ISSUE THREAD

**Issue by moddyz** [AcademySoftwareFoundation/openvdb#774]
Hi!

There is an old branch named `v3.3.0` on this remote.  There is also a tag named `v3.3.0`.

This results in errors when running certain git operations, such as trying to push the ref `v3.3.0` to another remote.

Is the `v3.3.0` branch still serving a purpose?  If so - could it be renamed, and if not, could it be removed?

Thanks!

**Comment by Idclip** [comment]
Hey @moddyz, good point, I don't see why we need both. @danrbailey can you remove the branch? Can touch on this in TSC if necessary

**Comment by danrbailey** [comment]
Thanks for bringing this up. In fact the v3.3.0 branch and tag were not on the same commit. I've deleted the v3.3.0 branch and moved the v3.3.0 tag to where the branch used to be (believing the branch position to be correct).

I also adjusted our branch protection rules - in future we would put down a v3.3 branch to track any future development based off this release and then put down v3.3.0, v3.3.1 tags when doing a release. I believe that was the original intention here.
